modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Sorting.tsx

```diff
@@ -401,15 +399,12 @@ const Sorting = ({
 					)
 					.filter(Boolean) as IFDSSort[];
 
-				if (storedFDSSorts.length > fdsSortsOrderArray.length) {
-					notOrdered = storedFDSSorts.filter(
-						(filter) =>
-							!fdsSortsOrderArray.includes(String(filter.id))
-					);
-				}
+				notOrdered = storedFDSSorts.filter(
+					(filter) => !fdsSortsOrderArray.includes(String(filter.id))
+				);
```

Comment:
This is a bugfix of a case I found by accident. It is possible that the ordered CSV contains a lot of values, and a smaller amount of saved value(s) are not in it. This happens when you delete items externally. 

Steps:
1. Create few actions in DSM UI. 
2. Save in any order.
3. Delete all action items through published "FDS Action" Objects page.
4. Create an item in DSM UI.

What happens:
- New item is not displayed. Length check in this line does not include it.

I applied all ordering fixes to Sorts, Filters and Actions.